spec/unit/resource/api600/c7000/storage_system_spec.rb

```diff
@@ -1,309 +1,16 @@
 require 'spec_helper'
 
-RSpec.describe OneviewSDK::API500::C7000::StorageSystem do
+RSpec.describe OneviewSDK::API600::C7000::StorageSystem do
   include_context 'shared context'
 
-  let(:client) { @client_500 }
+  let(:client) { @client_600 }
   let(:empty_item) { described_class.new(client) }
 
   it 'the UNIQUE_IDENTIFIERS shoud have name, uri and hostname' do
     expect(described_class::UNIQUE_IDENTIFIERS).to eq(%w[name uri hostname])
   end
 
```

Comment:
Is this test required?

Reply:
No. removing it. Thanks for finding it out.